src/_P023_OLED.ino

```diff
@@ -165,22 +172,29 @@ boolean Plugin_023(byte function, struct EventStruct *event, String& string)
       {
         if (Settings.TaskDevicePin3[event->TaskIndex] != -1)
         {
+          int index = Settings.TaskDevicePluginConfig[event->TaskIndex][0] == 0x3C
+                    ? 0
+                    : 1;
           if (!digitalRead(Settings.TaskDevicePin3[event->TaskIndex]))
           {
-            Plugin_023_displayOn();
-            displayTimer = Settings.TaskDevicePluginConfig[event->TaskIndex][2];
+            Plugin_023_displayOn(OLED_Settings[index]);
+            OLED_Settings[index].displayTimer = Settings.TaskDevicePluginConfig[event->TaskIndex][2];
           }
         }
         break;
       }
 
     case PLUGIN_ONCE_A_SECOND:
       {
-        if ( displayTimer > 0)
+        int index = Settings.TaskDevicePluginConfig[event->TaskIndex][0] == 0x3C
+          ? 0
+          : 1;
+
+        if (OLED_Settings[index].displayTimer > 0)
```

Comment:
displayTimer may not be initialized here when plugin is not yet gone through the init routine..

Reply:
it's initialized [here](https://github.com/workgroupengineering/ESPEasy/blob/1cc23bcee7bd805d65cf6e3045671466834023d0/src/_P023_OLED.ino#L164)